templates/ibmcloud-powervs-doc.md

```diff
@@ -0,0 +1,150 @@
+### Provision local boostrap management cluster:
+
+1. Create simple, local bootstrap cluster with a control-plane and worker node
+
+    Using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/):
+
+    ```console
+    ~ kind create cluster --name my-bootstrap --config bootstrap.yaml
+    ```
+
+    Example bootstrap.yaml:
+    ```yaml
+    kind: Cluster
+    apiVersion: kind.x-k8s.io/v1alpha4
+    nodes:
+       - role: control-plane
+       - role: worker
+    ```
+
+    Make sure the nodes are in `Ready` state before moving on.
+
+    ```console
+    ~ kubectl get nodes
+    NAME                         STATUS   ROLES                  AGE   VERSION
+    my-bootstrap-control-plane   Ready    control-plane,master   46h   v1.20.2
+    my-bootstrap-worker          Ready    <none>                 46h   v1.20.2
+    ```
+
+2. Set workload cluster environment variables
+
+    Make sure these value reflects your API Key for PowerVS environment in IBM Cloud.
+
+    ```console
+    export IBMCLOUD_API_KEY=<YOUR_API_KEY>
+    ```
+
+3. Initialize local bootstrap cluter as a management cluster
+
+    This cluster will be used to provision a workload cluster in IBM Cloud.
+
+    ```console
+    ~ clusterctl init --infrastructure ibmcloud:<TAG>
+    ```
+
+    Output:
+    ```console
+    Fetching providers
+    Installing cert-manager Version="v1.5.3"
+    Waiting for cert-manager to be available...
+    Installing Provider="cluster-api" Version="v0.4.4" TargetNamespace="capi-system"
+    Installing Provider="bootstrap-kubeadm" Version="v0.4.4" TargetNamespace="capi-kubeadm-bootstrap-system"
+    Installing Provider="control-plane-kubeadm" Version="v0.4.4" TargetNamespace="capi-kubeadm-control-plane-system"
+    Installing Provider="infrastructure-ibmcloud" Version="v0.1.0-alpha.2" TargetNamespace="capi-ibmcloud-system"
+
+    Your management cluster has been initialized successfully!
+
+    You can now create your first workload cluster by running the following:
+
+    clusterctl generate cluster [name] --kubernetes-version [version] | kubectl apply -f -
+    ```
+
+4. Provision workload cluster in IBM Cloud PowerVS
+
+    You can use clusterctl to render the yaml through templates.
+
+    **Note:** the `IBMPOWERVS_IMAGE_ID` value below should reflect the ID of the custom qcow2 image
+
+    ```console
+    IBMPOWERVS_SSHKEY_NAME="my-pub-key" \
+    IBMPOWERVS_VIP="192.168.151.22" \
+    IBMPOWERVS_VIP_EXTERNAL="158.175.162.22" \
```

Comment:
Might be better to add how the assigned VIP is generated here

Reply:
Agree with @Amulyam24 . If the doc link of PowerVS on IBM Cloud can be added as well it should be helpful.